features/admin.rules.v1/data.ts

```diff
@@ -0,0 +1,280 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
```

Comment:
```suggestion
 * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
```

Reply:
That's an extension rename of an existing file.